man/systemd.exec.xml

```diff
@@ -81,6 +81,10 @@
   <refsect1>
     <title>Paths</title>
 
+    <para>The following settings may be used to change a service's view of the filesystem. Please note that all paths
+    must be fully normalized, i.e. no component of the path may be <literal>.</literal> or <literal>..</literal>.
```

Comment:
This is not true, since 2f4d31c1173e51ea40631bf1bb9b15e01ab318e9 or thereabouts.

Reply:
More precisely: the path must be absolute and cannot contain `..`, but `.` and repeated slashes and such are OK.